content/en/docs/collector/configuration.md

```diff
@@ -30,14 +30,22 @@ example:
 otelcol --config=customconfig.yaml
 ```
 
-You can also provide configurations using environment variables, YAML paths, or
-HTTP URIs. For example:
+You can also provide configurations using environment variables, HTTP URIs, or
+YAML paths. For example:
 
 ```shell
 otelcol --config=env:MY_CONFIG_IN_AN_ENVVAR --config=https://server/config.yaml
 otelcol --config="yaml:exporters::debug::verbosity: normal"
 ```
 
+{{% alert title="Tip" color="primary" %}}
+
+When referring to nested keys in YAML paths, make sure to use double colons (::)
+to avoid confusion with namespaces that contain dots. For example:
+`receivers::docker_stats::metrics::container.cpu.utilization::enabled: false`.
+
```

Comment:
Do we want this as advice to the user or as an explanation of what we are doing? (Genuine question)

Reply:
I like the "tip" approach but I would also be OK with making it more of an explanation. I have no strong preference to be honest.